src/ui/popup.js

```diff
@@ -272,7 +272,12 @@ class Popup extends Evented {
             }
         }
 
-        const offsetedPos = this._pos.add(offset[anchor]).round();
+        let offsetedPos = this._pos.add(offset[anchor]);
+        //snap to interger offset if this is last tick of an easing
+        //animation, or if this event didn't come from easing loop
+        if (ev && (ev.finalEasingTick === true || ev.easing !== true)) {
```

Comment:
Thanks @jfirebaugh . It should work, I will look at folding that in after @mourner posts his feedback.